src/CharacterHups.jsx

```diff
@@ -1,13 +1,13 @@
 // import * as THREE from 'three';
-import React, {useState, useEffect, useRef} from 'react';
-import classnames from 'classnames';
-import dioramaManager from '../diorama.js';
-import {RpgText} from './RpgText.jsx';
-import styles from './CharacterHups.module.css';
+import React, { useState, useEffect, useRef } from "react";

```

Comment:
Please Make sure you keep the curlies and quotes the same as they were, and follow the format for these for additions as well. Really hard to read the PR when there is so much changed that is just formatting.